crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -47,6 +48,7 @@ impl CommandBase {
             args,
             #[cfg(test)]
             global_config_path: None,
+            global_auth_path: None,
```

Comment:
This field is getting shipped in release if it isn't prefixed with `#[cfg(test)]`
```suggestion
            #[cfg(test)]
            global_auth_path: None,
```
To avoid future mistakes like this, we might want to put these a new struct e.g.:
```
#[cfg(test)]
struct ConfigTestOverrides {
    global_config_path: Option<AbsoluteSystemPathBuf>,
    global_auth_path: Option<AbsoluteSystemPathBuf>,
}
...
#[cfg(test)]
pub overrides: Option<ConfigTestOverrides>
```

Reply:
I'll think about how to handle backwards compat and/or if we're ok with invalidating a previous login some more before merging / committing to this in particular.